### PR TITLE
Use file scoped namespaces

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/71192

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-compiler ptal.  Thanks!

**Review by RikkiGibson — APPROVED** [review]

File: src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs [hunk 1]
```diff
@@ -45,38 +45,31 @@ private void WriteInternal()
         {
             WriteFileHeader();
 
-            WriteLine("namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax");
-            OpenBlock();
-            WriteLine();
```
[the remark this answers is not in the document]
Comment (CyrusNajmabadi) [reply src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs:50]: yes.  that line was intended to be deleted.  this just looks weird for c#:

```C#
namespace Foo
{

    class Bar
```

[the remark this answers is not in the document]
Comment (CyrusNajmabadi) [reply src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs:50]: Note: WriteGreenTypes adds its own starting line.  so we'll get:

```C#
namespace X;

class Y
```

which is waht we'd want.

[the remark this answers is not in the document]
Comment (RikkiGibson) [reply src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs:50]: Consider deleting the corresponding line in WriteSyntax. Thanks

[the remark this answers is not in the document]
Comment (CyrusNajmabadi) [reply src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs:50]: done.

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 3)

**Pull request merged**